autodoc/writecommands.ts

```diff
@@ -12,9 +12,9 @@ layout: page
 title: Commands
 permalink: pages/commands
 ---
-Here is a list of all of the commands available for Sage, with the format \`s;command [arguments]\`.
+Here is a list of all of the commands available for Sage, usable with \`/command\`.
```

Comment:
```suggestion
Here is a list of all of the commands available for Sage, usable with \`/<command name here>\`.
```
I feel as if this clarifies the new system better for people & staff entirely new to discord.